docs/scripting/functions/SetObjectMaterialText.md

```diff
@@ -53,7 +53,7 @@ Color embedding can be used for multiple colors in the text.
 :::warning
 
 You MUST use ARGB color format, not RGBA like used in client messages etc.
-
+Text does not updates after 16 times this function is used.
```

Comment:
```suggestion
Text does not update after 16 calls against the same object.
```

Reply:
Also, is there a source for this? I've not heard of this behaviour before.

Reply:
Tested it out, can confirm - after the 16th change it stops updating.

Code used: https://pastebin.com/SmUw813m
deps: YCMD

Did it both manually, slowly and through a loop. Both stopped updating after 16. The object still gets changed (as in you can visibly see that it refreshes) but doesn't change.

Reply:
Oh and can confirm another bug. Once this happens, the result of **SetObjectMaterialText** is still **1** aka OK, not **0**, as mentioned in the **Returns** section

Adjusted code to print result of the function: https://pastebin.com/xMsh5Kge

Reply:
Thanks for confirming that!

Reply:
Works well with PlayerObject tho